spec/features/users/index_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'user pages' do
+  let(:roster) { create :roster }
```

Comment:
I believe rubocop disagrees, I could be wrong though.

Reply:
@mboneil10 you do. When you don't need it is:

```ruby
let :roster do
  create :roster
end
```

But with no parentheses using curly braces you'll get a syntax error - essentially because curly braces are more ambiguous than `do` and `end`.

Reply:
Whoops got it!